action.yml

```diff
@@ -10,7 +10,7 @@ inputs:
     description: Image name (including registry)
   imageTag:
     required: false
-    description: Image tag (defaults to latest)
+    description: One or more comma-separated image tags (defaults to latest)
```

Comment:
Would it make sense to make it a string OR and array of strings? (To make use of YAML syntax for the list.)

Reply:
Is that possible? I thought action inputs can only be strings because they get stores as environment variables.

Reply:
Sounds good, I wasn't aware of the limitations here. Thanks!